client/app/auth/auth.js

```diff
@@ -1,6 +1,6 @@
 angular.module('throwcast.auth', [])
 
 .controller('AuthController', function($scope, $http){
-  $scope.signup = 'Sign-Up';
-  $scope.signin = 'Sign-In';
+  $scope.signup = 'Signup';
```

Comment:
Actually looks like you are trying to test if the controller is linking up to the view. You can ignore my last comment.